react-stopwatch-component/src/stopwatch.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+
+export default class StopWatch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { start: false, ticks: 0 };
+    this.playPause = this.playPause.bind(this);
+    this.reset = this.reset.bind(this);
+  }
+
+  tick() {
+    this.setState({
+      ticks: this.state.ticks + 1
+    });
+  }
+
+  playPause() {
+    const play = this.state.start;
+    if (play === false) {
+      this.intervalId = setInterval(() => this.tick(), 1000);
+      this.setState({
+        start: true
+      });
+    } else {
+      clearInterval(this.intervalId);
+      this.setState({
+        start: false
+      });
+    }
+  }
+
+  reset() {
+    const play = this.state.start;
+    if (play === false) {
+      this.setState({
+        ticks: 0
+      });
+    }
+    clearInterval(this.intervalId);
+  }
```

Comment:
Clicking the stopwatch face shouldn't pause the timer.

Reply:
<img width="304" alt="Screen Shot 2022-09-04 at 12 46 19 PM" src="https://user-images.githubusercontent.com/97363006/188331452-051ba74f-fb49-4e83-8720-3470b1e734cc.png">